fabric-networking-v0/src/main/java/net/fabricmc/fabric/api/server/PlayerStream.java

```diff
@@ -43,30 +43,24 @@
 	private PlayerStream() { }
 
 	public static Stream<ServerPlayerEntity> all(MinecraftServer server) {
-		if (server.getPlayerManager() != null) {
-			return server.getPlayerManager().getPlayerList().stream();
-		} else {
-			return Stream.empty();
-		}
+		return server.getPlayerManager() != null ? server.getPlayerManager().getPlayerList().stream() : Stream.empty();
```

Comment:
We aren't writing patches, I don't think we need to go that crazy on the tenary operators.

Maybe just drop the `else` block and return `empty` like this:

```java
if (server.getPlayerManager() != null) {
	return server.getPlayerManager().getPlayerList().stream();
}

return Stream.empty();
```